app/payment-link-v2/amount/get-index.controller.test.js

```diff
@@ -73,10 +71,8 @@ describe('Amount Page - GET controller', () => {
       res = {}
       controller(req, res)
 
-      expect(responseSpy.called).to.equal(true)
-      expect(mockResponses.response.args[0]).to.include(req)
-      expect(mockResponses.response.args[0]).to.include(res)
-      expect(mockResponses.response.args[0]).to.include('amount/amount')
+      sinon.assert.called(responseSpy)
```

Comment:
Don't need to check it's `called` as the `calledWith` asserts that as well as checking the arguments.